models/jules/R/model2netcdf.JULES.R

```diff
@@ -32,19 +32,19 @@ model2netcdf.JULES <- function(outdir) {
                 row.names = TRUE,
                 quote = FALSE)
     
-    vars <- ncdf4.helpers::nc.get.variable.list(nc)
+    vars <- names(nc[['var']])
     # Check that frac is reported
     if("frac_grid" %in% vars){
       frac <- ncdf4::ncvar_get(nc, "frac_grid")
     } else {
       PEcAn.logger::logger.warn("Surface type fraction is not an output and thus other outputs may not be parseable")
     }
     
-    base_dims <- ncdf4.helpers::nc.get.dim.names(nc, "GPP") # This isn't the best example, except that GPP is the default with read.output (and it is not reported by surface type)
+    base_dims <- nc$var[["GPP"]]$dim[[1]]$name # This isn't the best example, except that GPP is the default with read.output (and it is not reported by surface type)
     for(i in seq_along(vars)){
       var <- vars[i]
       nonstd_var <- nrow(PEcAn.utils::standard_vars[which(PEcAn.utils::standard_vars$Variable.Name == var),]) == 0  
-      dims <- ncdf4.helpers::nc.get.dim.names(nc, var)
+      dims <- nc$var[[var]]$dim[[1]]$name
```

Comment:
See above. This will only return the name of the first dimension, but we need the names of _all_ dimensions.